src/components/Button.js

```diff
@@ -1,19 +1,28 @@
+/* eslint-disable react/button-has-type */
```

Comment:
* [ ] Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
- Do not make change to linter configs else you miss the main learning objective in relation to the rule.